### PR TITLE
set "supportability.atm_sampling_ratio" and "sampling.threshold" span…

### PR DESCRIPTION
**Pull request by samohte**
set "supportability.atm_sampling_ratio" and "sampling.threshold" span attributes as specified.

### CODE REVIEW

File: source/extensions/tracers/opentelemetry/samplers/dynatrace/dynatrace_sampler.cc [hunk 1]
```diff
@@ -73,6 +72,26 @@ class DynatraceTag {
   uint32_t path_info_;
 };
 
+// add Dynatrace specific span attributes
+void addSamplingAttributes(uint32_t sampling_exponent,
+                           std::map<std::string, std::string>& attributes) {
+
+  const auto multiplicity = SamplingState::toMultiplicity(sampling_exponent);
+  // The denominator of the sampling ratio. If for example the Dynatrace OneAgent samples with a
```
Comment (joaopgrassi) [inline source/extensions/tracers/opentelemetry/samplers/dynatrace/dynatrace_sampler.cc]:
```suggestion
  // The denominator of the sampling ratio. If, for example, the Dynatrace OneAgent samples with a
```

It's a bit hard to understand this, maybe we are missing some dots and new sentences?

File: source/extensions/tracers/opentelemetry/samplers/dynatrace/dynatrace_sampler.cc [hunk 2]
```diff
@@ -73,6 +72,26 @@ class DynatraceTag {
...
+  // probability the value of supportability.atm_sampling_ratio would be 16
```
Comment (joaopgrassi) [inline source/extensions/tracers/opentelemetry/samplers/dynatrace/dynatrace_sampler.cc]:
```suggestion
  // probability the value of supportability.atm_sampling_ratio would be 16
```
Hum why would it be 16? Not sure if I follow it

File: source/extensions/tracers/opentelemetry/samplers/dynatrace/dynatrace_sampler.cc [hunk 3]
```diff
@@ -73,6 +72,26 @@ class DynatraceTag {
...
+  // ratio is also called multiplicity
```
Comment (joaopgrassi) [inline source/extensions/tracers/opentelemetry/samplers/dynatrace/dynatrace_sampler.cc]:
```suggestion
  // . Note: Ratio is also known as multiplicity.
```

I think this is just a "note" right? I'd start this in its own sentence, or add a final . before it

**Review by joaopgrassi — APPROVED** [review]

File: source/extensions/tracers/opentelemetry/samplers/dynatrace/dynatrace_sampler.cc (hunk 3 above)
Comment (samohte) [reply source/extensions/tracers/opentelemetry/samplers/dynatrace/dynatrace_sampler.cc]: I partially copied the description from https://bitbucket.lab.dynatrace.org/projects/DEUS/repos/semantic-dictionary/browse/doc/fields/supportability.md
Obviously, I removed essential parts. I modified the comment.

**Pull request merged**